content/issues/tracking-your-work-with-issues/using-issues/linking-a-pull-request-to-an-issue.md

```diff
@@ -19,16 +19,16 @@ topics:
 shortTitle: Link PR to issue
 ---
 
-> [!NOTE]
-> The special keywords in a pull request description are interpreted when the pull request targets the repository's _default_ branch. However, if the PR's base is _any other branch_, then these keywords are ignored, no links are created and merging the PR has no effect on the issues. **If you want to link a pull request to an issue using a keyword, the PR must be on the default branch.**
-
 ## About linked issues and pull requests
 
-You can link an issue to a pull request manually or using a supported keyword in the pull request description.
+You can link an issue to a pull request manually or using a supported keyword in the pull request description, that is, the summary text added by the author when they created the pull request.
```

Comment:
@MichaelChirico - I updated your original definition to bring it into line with our style guide. Is this clear to you?

Reply:
Looks great, thanks for the refinement!